cli/src/pcluster/resources/batch/docker/build-docker-images.sh

```diff
@@ -34,10 +34,10 @@ build_docker_image() {
 
 if [ -z "${IMAGE}" ]; then
     echo "No image to build specified. Building all images..."
-    for file in $(find `pwd` -type f -name Dockerfile); do
+    find "$(pwd)" -type f -name Dockerfile -exec sh -c '
         IMAGE_TAG=$(dirname "${file}" | xargs basename)
-        build_docker_image "${IMAGE_TAG}"
-    done
+        push_docker_image "${IMAGE_TAG}"
```

Comment:
Should be `build_docker_image`

Reply:
Good catch, I fixed + changed the loop a little after doing more testing